### ISSUE TITLE
Em falta linha 4600 na paragem 090202 Baixa Banheira (EN 11 176A) 

### ISSUE THREAD

**Issue by hmaichat** [carrismetropolitana/gtfs#393]
**Descreva o erro**  
A linha 4600 sentido Barreiro para na paragem #090202 apesar de assim não estar descrito na página. O mesmo acontece com a linha 4601, de menos importância já que vai ser removida em agosto. 

**Área(s) afetada(s)**  
Quais as áreas afetadas pelo erro identificado:

- [ ] Área 1 (Viação Alvorada)
- [ ] Área 2 (Rodoviária de Lisboa)
- [ ] Área 3 (TST)
- [x] Área 4 (Alsa Todi)

**Valor(es) esperado(s)**  
Se possível, uma descrição clara e concisa da situação sem erro / situação corrigida.

**Screenshots**  
![image](https://github.com/user-attachments/assets/717f52a8-24b8-46c9-8226-07306d8265e3)

**Contexto adicional**  
Adicione qualquer outra informação útil à resolução do problema.

**Comment by inesclemente** [comment]
Olá @hmaichat, obrigada pelo alerta. A situação já se encontra resolvida.

**Comment by LaPingvino** [comment]
Esta paragem só aparece na variante Chão Duro, não no resto do dia. O autocarro sempre pare aí sim. Ainda há de ser corrigido na GTFS.

**Comment by hmaichat** [comment]
> Esta paragem só aparece na variante Chão Duro, não no resto do dia. O autocarro sempre pare aí sim. Ainda há de ser corrigido na GTFS.

Sim, quando escrevi isto a linha ainda era a 4603. O problema manteve-se, mas já deve estar resolvido por Abril
